packages/game-bridge/README.md

```diff
@@ -1,27 +1,37 @@
-<div align="center">
-  <p align="center">
-    <a href="https://docs.x.immutable.com/docs">
-      <img src="https://cdn.dribbble.com/users/1299339/screenshots/7133657/media/837237d447d36581ebd59ec36d30daea.gif" width="280"/>
-    </a>
-  </p>
-  <h1>Immutable Game Bridge</h1>
-</div>
-
-## How to Create Index Files for Immutable Unity and Unreal SDK
-
-1. In the root directory of this repository, run:
+# Game Bridge
+
+### Pre-requisites
+
+Install dependencies for the workspace
+
 ```bash
 yarn install
 ```
-2. Navigate to the game-bridge directory:
+
+### Quick Start Guide
+
+All commands below need to be run in the context of the `game-bridge` package where this README is located. Read more about context [here](../../../README.md#context).
+
+Building the Game Bridge for Unity and Unreal:
+
 ```bash
-`cd packages/game-bridge`
+yarn build
 ```
-3. Build the files:
+
+Note: The above build will bundle the index files in the following locations (relative to this README):
+- Unity: `./dist/unity/index.html`
+- Unreal: `./dist/unreal/index.js`, `./dist/unreal/index.js.map`
+
+Linting:
+
 ```bash
-yarn build
+yarn lint
 ```
 
-The bundled index files will be generated in the following locations:
-- Unity: `game-bridge/dist/unity/index.html`
-- Unreal: `game-bridge/dist/unreal/index.js`
+### About
+
+The Game Bridge is a package that provides a way to interact with the Typescript SDK from Unity and Unreal by wrapping around the entire SDK, outputting it in an (`index.js` & `index.js.map`)/`index.html` file that can then be imported into Unreal/Unity respectively.
+
+[Read more about how the game-bridge works here](https://immutable.atlassian.net/wiki/spaces/GSK/pages/2307129395/Game+SDK+Overview)
```

Comment:
This is a public repository, but we're linking to an internal document. Perhaps we should consider linking to a section like [this](https://docs.immutable.com/docs/zkEVM/sdks/unity/faq#i-use-a-custom-engine-rather-than-unity-or-unreal-how-can-i-integrate-with-immutable) instead?